aten/src/ATen/native/TensorShape.cpp

```diff
@@ -4090,4 +4091,11 @@ int64_t dense_dim_default(const Tensor& self) {
   return self.dim();
 }
 
+Tensor chunk_backward(at::TensorList grads, const at::Tensor & input, int64_t chunks, int64_t dim) {
+    // Not implemented
+    // chunk_backward is only used by Nested Jagged Tensor.
+    TORCH_CHECK_NOT_IMPLEMENTED(input.is_nested() && input.layout() == c10::kJagged, "chunk_backward is not implemented explicitly");
```

Comment:
nit: should this always assert? if it's a NJT then I would imagine it should directly go to the ops.py implementation, otherwise that seems unexpected

Reply:
This assert is for NST.

Reply:
right, but even in the case where input _is_ NJT, it seems like we should still throw an exception? (which doesn't happen now I think)

Reply:
nit: Maybe `TORCH_INTERNAL_ASSERT` instead? `TORCH_CHECK_NOT_IMPLEMENTED` implies that we'd want to eventually implement it.